policyengine_us/variables/gov/irs/income/taxable_income/adjusted_gross_income/above_the_line_deductions/student_loan_interest_ald.py

```diff
@@ -0,0 +1,76 @@
+from policyengine_us.model_api import *
+
+
+class student_loan_interest_ald(Variable):
+    value_type = float
+    entity = Person
+    label = "Student loan interest ALD"
+    unit = USD
+    definition_period = YEAR
+    documentation = "Above-the-line deduction for student loan interest"
+    reference = "https://www.law.cornell.edu/uscode/text/26/221"
+
```

Comment:
to avoid in formula
```suggestion
    defined_for = "student_loan_interest_ald_eligible"

```
and make a new variable that defines it for:
* is_tax_unit_head_or_spouse
* separate exclusion
* eligible student status (points to AOC/LLC eligibility)